src/server/scripts/Outland/Auchindoun/SethekkHalls/sethekk_halls.h

```diff
@@ -20,14 +20,30 @@
 
 #include "CreatureAIImpl.h"
 
+#define DataHeaders "SH"
+
+constexpr uint32 EncounterCount = 3;
```

Comment:
```suggestion
uint32 constexpr EncounterCount = 3;
```
https://www.azerothcore.org/wiki/cpp-code-standards#defining-const-variables

Reply:
![image](https://user-images.githubusercontent.com/47818697/219880915-ae1ab601-e2d4-4524-85fe-57388e4dcc91.png)